### PR TITLE
Fix for issue #38

### PR DESCRIPTION
**Pull request by katarzyna-z**

### CODE REVIEW

File: influx/influx.go [hunk 1]
```diff
@@ -153,12 +153,21 @@ func (f *influxPublisher) Publish(contentType string, content []byte, config map
 		for k, v := range m.Tags() {
 			tags[k] = v
 		}
+
+		// NOTE: uint64 is specifically not supported by influxdb client due to potential overflow
+		//without convertion of uint64 to int64, data with uint64 type will be saved as strings in influx database
+		data := m.Data()
+		v, ok := m.Data().(uint64)
+		if ok {
+			data = int64(v)
```
Comment (andrzej-k) [inline influx/influx.go]: Any number bigger than ^uint64(0)/2 will be a negative number - consider adding such check and logging such cases

File: influx/influx.go [hunk 2]
```diff
@@ -153,12 +153,24 @@ func (f *influxPublisher) Publish(contentType string, content []byte, config map
...
+			if v > (^uint64(0) / 2) {
```
Comment (andrzej-k) [inline influx/influx.go]: consider having this as const

**Comment by andrzej-k** [comment]
LGTM

**Comment by geauxvirtual** [comment]
Fixes or features to plugins should increment the plugin version.

**Pull request merged**